collectors/proc.plugin/README.md

```diff
@@ -322,6 +323,47 @@ each state.
 
 By default Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after Netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though).
 
+### Monitoring wireless network interfaces
+
+The settings for monitoring wireless is in the `[plugin:proc:/proc/net/wireless]` of your `netdata.conf` file.
+
+```conf
+    enable new interfaces detected at runtime = yes
+    status for all interfaces = yes
+    quality for all interfaces = yes
+    discarded packets for all interfaces = yes
+    missed for all interfaces = yes
```

Comment:
missed what?

Reply:
I renamed it to `missed beacon for all interfaces`.